### PR TITLE
Km sprint 4 organize and lint

### PR DESCRIPTION
**Pull request by Kristiannmiller**
## Description
This PR:
- Creates organizational headers in scripts.js and domUpdates.js for easy navigation through codebase
- Rearranges functions in scripts.js and domUpdates.js so that related functions are nearby and easy to find.
- Renames non-descriptive function names for developer empathy
- Suggests changes to the Recipe class in order for it to be utilized in a more functional way
- Utilizes the instantiation of Recipes in the global variable (recipeData/recipes) so that the Recipe methods can be used throughout the codebase.
- Fixes small lint errors throughout the codebase

What still needs done for the final refactor?
- addToFavorites function: Turn into a handler function using some of the functionality seen in domUpdates.addToMyRecipes as a guide. This function name should also probably be changed to be more descriptive since it's actually delegating where the click event on main came from, not JUST adding to favorites.
- Utilize the Recipe class methods
- Use "display" instead of show in function names across the codebase
- Look into the way we're using two recipe global variables. Can we simplify?

## Motivation and Context
scripts.js and domUpdates.js were difficult to navigate, and the function names were often vague or unclear. This PR begins to fix this issue. This also closes the lint issue in sprint 4.

## How Has This Been Tested?
Tests have been run in terminal using spies and chai
Changes have been tested via the browser console

## Types of changes
- [X] Lint

## Checklist:
- [X] My code follows the code style of this project.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: src/domUpdates.js [hunk 1]
```diff
@@ -191,21 +183,11 @@ let domUpdates = {
...
-    } else if (event.target.id === "exit-recipe-btn") {
-      this.exitRecipe();
-    } else if (this.isDescendant(event.target.closest(".recipe-card"), event.target)) {
-      this.openRecipeInfo(event, recipeData, ingredientsData);
-    }
+  // ************ HELPERS *************** //
+  capitalize(words) {
+    return words.split(" ").map(word => {
+      return word.charAt(0).toUpperCase() + word.slice(1);
+    }).join(" ");
```
Comment (ericberglund117) [inline src/domUpdates.js:190]: Much more organized now. I was curious about the 164-173 indentation earlier. Thank you for updating that

File: src/scripts.js [hunk 2]
```diff
@@ -76,121 +75,138 @@ function getRecipes() {
...
+    .then(data => console.log(data))
+    .catch(error => console.log(error))
+}
+
+// ************ RECIPES *************** //
+function instantiateRecipes(recipes) {
+  recipes = recipes.map(recipe => new Recipe(recipe));
+  return recipes
+}
+
```
Comment (ericberglund117) [inline src/scripts.js:108]: I like how you moved this Fetch post request up to display when it is being called. It makes more sense this way

File: src/user.js [hunk 3]
```diff
@@ -101,7 +101,6 @@ class User {
       }
     })
   }
-
```
Comment (ericberglund117) [inline src/user.js:104]: good, I always hated that sneaky hyphen

**Review by ericberglund117 — APPROVED** [review]

File: src/scripts.js (hunk 2 above)
Comment (chadrick-d-dev) [reply src/scripts.js:108]: I like the new variable names you used. Demonstrates more developer empathy.

File: src/user.js (hunk 3 above)
Comment (chadrick-d-dev) [reply src/user.js:104]: Yes, love that you got rid of that meddling hyphen!

**Review by chadrick-d-dev — APPROVED** [review]
Great changes Kristi! I especially appreciated how you renamed some of the functions and variables.

**Pull request merged**